pkg/tests/cross-tests/pretty.go

```diff
@@ -72,6 +72,9 @@ func (s prettyValueWrapper) GoString() string {
 		tL := tp.TypeReferenceString(v.Type())
 		indent := strings.Repeat("  ", level)
 		switch {
+		case v.IsNull():
+			// TODO
```

Comment:
What does this signify? Thanks!

Reply:
I was under the impression tftypes.NewValue(typ, nil) didn't work but after your comment I tried it again and it works. My bad, thank you!